### PR TITLE
Added-Test-For-Settings-Route

### LINKED ISSUE #182: Add test for routing in UserCard
Add a test, that make sure that the linking in UserCard actually works. It could look something like this
```jsx
it("should redirect to settings", () => {
  render(
    <MemoryRouter>
      <UserCard
        user={{
          firstName: "John",
          lastName: "Doe",
          email: "john@doe.com",
        }}
      />
      <Route path="/settings">Only visible when on /settings</Route>
    </MemoryRouter>
  );
  /* code to expand the list and click on the settings link */
  expect(
    screen.queryByText("Only visible when on /settings")
  ).toBeInTheDocument();
});
```

### PR DESCRIPTION
**Pull request by ruaridhm**
Closes #182

### CODE REVIEW

File: src/components/Sidebar/components/UserCard/UserCard.test.js [hunk 1]
```diff
@@ -71,3 +71,29 @@ it("should collapse when clicked again", () => {
...
+      />
+      <Route path="/settings">Only visible when on /settings</Route>
+    </MemoryRouter>
+  );
+
+  userEvent.click(screen.queryByText(testUser.firstName));
+  userEvent.click(screen.queryByText("Profile & Settings"));
+  expect(
+    screen.queryByText("Only visible when on /settings")
+  ).toBeInTheDocument();
```
Comment (MoSattler) [inline src/components/Sidebar/components/UserCard/UserCard.test.js:98]: Small thing. Can you add one check at the beginning that makes sure that the text us not visible before clicking all the things?

**Comment by MoSattler** [comment]
Why did you revert the tests?

**Comment by ruaridhm** [comment]
Apologies. That was an accidental revert. I was reverting something in an other PR and must have done it here too. I was in a rush before heading to work! I'l pop it back up now

File: src/components/Sidebar/components/UserCard/UserCard.test.js [hunk 2]
```diff
@@ -71,3 +71,29 @@ it("should collapse when clicked again", () => {
...
+  render(
+    <MemoryRouter>
+      <UserCard
+        user={{
+          firstName: "John",
+          lastName: "Doe",
+          email: "john@doe.com",
+        }}
+      />
+      <Route path="/settings">Only visible when on /settings</Route>
```
Comment (MoSattler) [inline src/components/Sidebar/components/UserCard/UserCard.test.js:90]: Can you use `RoutePath` like [here](https://github.com/HandsOnLearning/the-project/blob/47d61c4af992edcc086a49486938cc11e197c909/src/components/Content/Content.tsx#L22)?

**Comment by MoSattler** [comment]
@limbovski we could, but that'd hardly couple the test for the user card to the settings page. Meaning, if we change something in the setttings page, the usercard test would break. 
We want to avoid this kind of coupling of component's tests.

**Pull request merged**